src/js/tabs/pid_tuning.js

```diff
@@ -481,31 +485,51 @@ TABS.pid_tuning.initialize = function (callback) {
+            $('.pid_tuning .ROLL input[name="dMinPitch"]').attr("max", "100");
+            $('.pid_tuning .PITCH input[name="p"]').attr("max", "200");
+            $('.pid_tuning .PITCH input[name="i"]').attr("max", "200");
+            $('.pid_tuning .PITCH input[name="d"]').attr("max", "200");
+            $('.pid_tuning .PITCH input[name="dMinPitch"]').attr("max", "100");
+            $('.pid_tuning .YAW input[name="p"]').attr("max", "200");
+            $('.pid_tuning .YAW input[name="i"]').attr("max", "200");
+            $('.pid_tuning .YAW input[name="d"]').attr("max", "200");
+            $('.pid_tuning .YAW input[name="dMinPitch"]').attr("max", "1000");
+            $('#sliderDMinRatio').attr({ "min": "0.5", "max": "1.5" });
+            $('#sliderDTermFilterMultiplier').attr({ "min": "0.5", "max": "1.5", "step": "0.025" });
+        }
+
         if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
             // Feedforward
             const feedforwardGroupCheck = $('input[id="feedforwardGroup"]');
-            const PID_FEEDFORWARD = (FC.ADVANCED_TUNING.feedforwardRoll ||
-                FC.ADVANCED_TUNING.feedforwardPitch ||
-                FC.ADVANCED_TUNING.feedforwardYaw);
-
-            feedforwardGroupCheck.prop('checked', PID_FEEDFORWARD !== 0);
+            const PID_FEEDFORWARD = FC.ADVANCED_TUNING.feedforwardRoll || FC.ADVANCED_TUNING.feedforwardPitch || FC.ADVANCED_TUNING.feedforwardYaw;
+            feedforwardGroupCheck.prop('checked', PID_FEEDFORWARD);
+            $('.feedforwardGroupCheckbox').addClass('switchery-disabled');
             $('select[id="feedforwardAveraging"]').val(FC.ADVANCED_TUNING.feedforward_averaging);
             $('input[name="feedforwardSmoothFactor"]').val(FC.ADVANCED_TUNING.feedforward_smooth_factor);
             $('input[name="feedforwardBoost"]').val(FC.ADVANCED_TUNING.feedforward_boost);
 
-            feedforwardGroupCheck.change(function() {
-                const checked = $(this).is(':checked');
-                $('.feedforwardGroup .suboption').toggle(checked);
-                if (!checked) {
-                    $('.pid_tuning .ROLL input[name="f"]').val(0);
-                    $('.pid_tuning .PITCH input[name="f"]').val(0);
-                    $('.pid_tuning .YAW input[name="f"]').val(0);
-                } else {
-                    $('.pid_tuning .ROLL input[name="f"]').val(FC.ADVANCED_TUNING.feedforwardRoll > 0 ? FC.ADVANCED_TUNING.feedforwardRoll : PID_DEFAULT[4]);
-                    $('.pid_tuning .PITCH input[name="f"]').val(FC.ADVANCED_TUNING.feedforwardPitch > 0 ? FC.ADVANCED_TUNING.feedforwardPitch : PID_DEFAULT[9]);
-                    $('.pid_tuning .YAW input[name="f"]').val(FC.ADVANCED_TUNING.feedforwardYaw > 0 ? FC.ADVANCED_TUNING.feedforwardYaw : PID_DEFAULT[14]);
-                }
-            }).change();
+            if (semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+                feedforwardGroupCheck.change(function() {
```

Comment:
This function is never used, can be removed.

Reply:
Revised legacy switches